app/templates/reminder-list/edit.hbs

```diff
@@ -0,0 +1,11 @@
+{{#with model as |reminder| }}
+<form class="create-reminder-form" >
+  <h2>Edit this reminder</h2>
+    {{ input value=reminder.title class="reminder-title-input" placeholder="Insert Title" }} <br>
+    {{ input value=reminder.date type="date" class="reminder-date-input" placeholder="Insert Date" }} <br>
+    {{ textarea value=reminder.notes class="reminder-notes-input" placeholder="Insert Note" }} <br>
+    {{#link-to "reminder-list.reminder-detail" reminder}}
+    <button class="reminder-submit-button" {{action "updateReminder" reminder.id on="click" bubbles="false"}}>Submit</button>
```

Comment:
I would like to go over this when you get in today primarily to talk about bubbles="false"